gamedata/alldefs_post.lua

```diff
@@ -1376,34 +1376,44 @@ function WeaponDef_Post(name, wDef)
 		if modOptions.shieldsrework == true then
 			local shieldCollisionExemptions = { --add the name of the weapons (or just the name of the unit followed by _ ) to this table to exempt from shield collision. 
 				'corsilo_', 'armsilo_', 'cormship_', 'armmship_', 'armthor_empmissile', 'armemp_', 'cortron_', 'corjuno_', 'armjuno_'
-			} 
-			local paralyzerShieldDamageMultiplier = 0.25 --change this multiplier to globally modify paralyzer damage to shields only.
+			}
+
+			-- For balance, paralyzers need to do reduced damage to shields, as their raw raw damage is outsized
+			local paralyzerShieldDamageMultiplier = 0.25
```

Comment:
consider 1/3. most paralyze weapons are very low range so a naive multiplier will be biased against them

Reply:
The actual value of this number is just balance, so I'm not too worried with the initial value. If we decide to go with shields are 100% immune to EMP, we set it to 0.